src/ui/components/DropdownMenu/styles.scss

```diff
@@ -0,0 +1,113 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+$link-font-size: 16px;
+
+.DropdownMenu {
+  @include text-align-start();
+
+  cursor: pointer;
```

Comment:
This actually creates a UX problem: the entire dropdown has a pointer cursor so there's no cursor style change on hover state of the actual link/button elements that are clickable versus the entire `DropdownMenu` which is not clickable. See this GIF:

![2017-09-07 13 22 56](https://user-images.githubusercontent.com/90871/30163172-dcea2972-93cf-11e7-9edf-c59a5448ff01.gif)

Because there's only a colour change on hover state this will be entirely unclear to colourblind users. Instead of adding a text decoration or something else on hover though just making only the actual clickable elements show the link cursor should do the trick.

Basically: this needs to work like the "Developer Hub" link does.

Reply:
I see, this should be fixed.